tests/scripts/task_python_microtvm.sh

```diff
@@ -28,6 +28,7 @@ make cython3
 run_pytest ctypes python-microtvm-zephyr-qemu_x86 tests/micro/zephyr --zephyr-board=qemu_x86
 run_pytest ctypes python-microtvm-zephyr-qemu_riscv32 tests/micro/zephyr --zephyr-board=qemu_riscv32
 run_pytest ctypes python-microtvm-zephyr-qemu_riscv64 tests/micro/zephyr --zephyr-board=qemu_riscv64
+run_pytest ctypes python-microtvm-zephyr-mps2_an521 tests/micro/zephyr --zephyr-board=mps2_an521
```

Comment:
cc @gromero is this board stable enough to run in CI? i thought we had I/O flakiness here

Reply:
Only enabling this to run one test for the CMSIS build. The rest are skipped

Reply:
oh oops, ignore. it's okay since your one test is build-only.